pkg/extensions/sync/utils.go

```diff
@@ -1,251 +1,40 @@
+//go:build sync
+// +build sync
+
 package sync
 
 import (
 	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
+	"io"
 	"os"
-	"path"
-	"regexp"
 	"strings"
-	"time"
 
-	"github.com/Masterminds/semver"
-	glob "github.com/bmatcuk/doublestar/v4"
-	"github.com/containers/common/pkg/retry"
 	"github.com/containers/image/v5/copy"
 	"github.com/containers/image/v5/docker"
-	"github.com/containers/image/v5/docker/reference"
 	"github.com/containers/image/v5/manifest"
-	"github.com/containers/image/v5/oci/layout"
+	"github.com/containers/image/v5/signature"
 	"github.com/containers/image/v5/types"
-	guuid "github.com/gofrs/uuid"
-	godigest "github.com/opencontainers/go-digest"
+	"github.com/docker/distribution/reference"
+	"github.com/opencontainers/go-digest"
```

Comment:
Let's keep `godigest` as an alias. It's less confusing when accessing the apis within the package.